.circleci/test-api.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+
+apt-get update
+apt-get install -y jq
+
+make serve-local &
```

Comment:
For test resilience, do not rely on default port and host: define at least `PORT` and use it both as a parameter to `serve-local` and in the tested `curl` target.

Reply:
Why not `openfisca serve`? 🙂

Reply:
If we store this in a separate shell script, we support running the tests outside from CI. I think it is great, but in this case we have to kill this process after it was created, otherwise the host machine will keep the process running.

Reply:
For some reason that I don't understand, `openfisca serve` does not work. Only `make serve-local` does, even though all it does is call `openfisca serve`. I tried restoring the cache from the `build` job in the `test_api` but it did not help.
Here are the [logs](https://app.circleci.com/pipelines/github/openfisca/country-template/105/workflows/2d3555e4-8e90-4998-af90-841596e7e44e/jobs/1086) `.circleci/test-api.sh: line 6: openfisca: command not found`. 
The job still succeeded but that is another problem that I will discuss below.

Reply:
The two other comments above are still valid 🙂